src/main/java/avd/inf/jdm/rentmycar/controller/AuthController.java

```diff
@@ -62,7 +62,7 @@ public ResponseEntity<Object> login(
     @PostMapping("/v1/auth/reset")
     public ResponseEntity<Object> resetPassword(   @RequestBody(required = false) AuthDTO authDTO){
         Optional<User> user = userService.getUserByEmail(authDTO.getEmail());
-        if(user.isEmpty()) {
+        if(user == null) {
```

Comment:
Heb je deze check bewust weer naar `==null` veranderd of had je de main niet gemerged in deze branch?